packages/ws-server/src/controllers/index.ts

```diff
@@ -81,6 +81,7 @@ const handleSendingRequestsToRelay = async ({
       chainId: (_, requestDetails) => [requestDetails],
       estimateGas: (params, requestDetails) => [...params, null, requestDetails],
       getStorageAt: (params, requestDetails) => [params[0], params[1], requestDetails, params[2]],
+      newFilter: (params, requestDetails) => [params[0], params[1], requestDetails, params[2], params[3]],
```

Comment:
Nice catch, there should be a test to confirm this and ensure no regressions in the future

Reply:
Good point. Added some unit tests